### ISSUE TITLE
Add the rule to warn about skipped heading levels

### ISSUE THREAD

**Issue by YusukeHirao** [markuplint/markuplint#1151]
Rel: https://github.com/whatwg/html/issues/8366

Correct:
```html
<h1>A</h1>
<h2>B</h2>
<h3>C</h3>
```

Incorrect:
```html
<h1>A</h1>
<h3>B</h3><!-- skipped heading levels -->
<h2>C</h2>
```

**Comment by kagankan** [comment]
Probably a duplicate of #695
